<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>position_mode_set_dialog</class>
 <widget class="QDialog" name="position_mode_set_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Joints mode set</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Choose foce mode joints:</string>
   </property>
   <widget class="QCheckBox" name="Joint8_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J8</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Joint3_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J3</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Joint5_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Joint1_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Joint4_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>40</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J4</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Joint2_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Joint7_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J7</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="Joint6_mode_choose_checkBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>70</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>J6</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="position_mode_apply_pushButton">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>101</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Set force(N):</string>
   </property>
   <widget class="QLineEdit" name="Constant_force_set_lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>139</x>
     <y>110</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>v/a_max set(mm/s)</string>
   </property>
   <widget class="QLineEdit" name="v_max_set_lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="a_max_set_lineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>v_max:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>a_max:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
